### PR TITLE
Integrate fav button with database

### PR DESCRIPTION
**Pull request by codydevloop**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -15,12 +15,16 @@ if (process.env.NODE_ENV === "production") {
...
-  process.env.MONGODB_URI ||
-  "mongodb+srv://azucena_1:wc5vclNd7KxqpytQ@cluster-p7dpjvmj.6prcb.mongodb.net/heroku_p7dpjvmj?retryWrites=true&w=majority"  
+mongoose.connect(process.env.MONGODB_URI || "mongodb://localhost/realestateapp", {
+  useNewUrlParser: true,
+  useFindAndModify: false,
+  useUnifiedTopology: true
+});
+// mongoose.connect(
+//   process.env.MONGODB_URI ||
+//   "mongodb+srv://azucena_1:wc5vclNd7KxqpytQ@cluster-p7dpjvmj.6prcb.mongodb.net/heroku_p7dpjvmj?retryWrites=true&w=majority"
```
Comment (azu20) [inline server.js:25]: need to uncomment lines 23-25, and comment lines 19-21

File: client/src/Pages/MainPage/Main.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import React, { Component } from "react";
-import API from "../../utils/API"
```
Comment (azu20) [inline client/src/Pages/MainPage/Main.js:2]: need to keep the import API from utils folder.

**Review by azu20 — CHANGES_REQUESTED** [review]
added 2 comments to review and update to complete merge request

**Comment by codydevloop** [comment]
several updates needed, will complete and request new merge

**Pull request closed without merging**